packages/loader/container-loader/src/connectionStateHandler.ts

```diff
@@ -178,46 +163,38 @@ export class ConnectionStateHandler extends EventEmitterWithErrorHandling<IConne
 
         const oldState = this._connectionState;
         this._connectionState = value;
+        let client: ILocalSequencedClient | undefined;
+        if (this._clientId !== undefined) {
+            client = this.handler.protocolHandler()?.quorum.getMember(this._clientId);
```

Comment:
put the quorum in a var, `const quorum = this.handler.protocolHandler()?.quorum` maybe also assert it's not undefined, or just return if undefined, not sure what the right behavir is

Reply:
Yes, we can add an assert here.

Reply:
Hmm, so I don't think we can either assert or return here because client can initiate a connection before protocol handler is initialized and we connect in read mode so we can get connected.